src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterface.cs

```diff
@@ -1248,7 +1206,7 @@ public override void WriteDebugLine(string message)
             else
             {
                 // NTRAID#Windows OS Bugs-1061752-2004/12/15-sburns should read a skin setting here...
```

Comment:
Is the comment still actual?

Reply:
I believe the comment is saying that if we decide to support color themes (aka skins), this is one place to put the code for Write-Debug.  Did a quick search for "read a skin" and found four places for Debug, Verbose, Warning, and ProgressPane.

Reply:
Thanks for clarify.

Closed.